job/job.go

```diff
@@ -79,6 +79,12 @@ type Job struct {
 	// Mime type pattern provided by the client
 	MimeType string `json:"mime_type"`
 
+	// Compute image size on supported mime-types
+	ExtractImageSize bool `json:"extract_image_size"`
```

Comment:
Why not make false by default here?

Reply:
`false` is the default value of `boolean` fields